apps/hardware/lib/hardware/model/component/cpu.ex

```diff
@@ -33,7 +33,7 @@ defmodule Helix.Hardware.Model.Component.CPU do
   end
 
   def create_from_spec(cs = %ComponentSpec{spec: spec}) do
-    cpu_id = PK.generate([0x0003, 0x0001, 0x0002])
+    cpu_id = PK.pk_for(__MODULE__)
```

Comment:
I don't like this, but let's leave it that way until we figure a better way to do that